pkg/awsutil/session.go

```diff
@@ -37,6 +37,7 @@ type Credentials struct {
 	SecretAccessKey string
 	SessionToken    string
 	AssumeRoleArn   string
+	ExternalId      string
```

Comment:
```suggestion
	ExternalID      string
```

I would kindly ask you to write `ID` [in uppercase](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). I know we are not consistent, but we try to :smile: